openmetadata-ui/src/main/resources/ui/src/pages/LineagePage/lineagePage.style.less

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+.lineage-page-container {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  height: 100%;
+
+  .ant-card-body {
+    height: 100%;
+    margin-right: 14px;
+    margin-left: 14px;
+  }
```

Comment:
Use size prop of card for margins
Probably className h-full for height

Reply:
updated